units/generate_tasks.go

```diff
@@ -76,8 +76,40 @@ func (j *generateTasksJob) generate(ctx context.Context, t *task.Task) error {
 		return nil
 	}
 
+	v, err := model.VersionFindOneId(t.Version)
```

Comment:
shouldn't this all be in the models package?

Reply:
Yeah, this is mostly just calling a function from the model package and then doing some error checking.